src/scripts/vanilla_video.js

```diff
@@ -37,7 +38,7 @@ export const onStorageChanged = async function (changes, areaName) {
 
 export const main = async function () {
   ({ defaultVolume } = await getPreferences('vanilla_video'));
-  pageModifications.register(`video:not([src], .${vanillaVideoClass})`, cloneVideoElements);
+  pageModifications.register(`${keyToCss('videoPlayer')} video:not([src], .${vanillaVideoClass})`, cloneVideoElements);
```

Comment:
targeting `videoPlayer` parents makes the `[src]` exclusion unnecessary here

```suggestion
  pageModifications.register(`${keyToCss('videoPlayer')} video:not(.${vanillaVideoClass})`, cloneVideoElements);
```